$OpenBSD: patch-source3_wscript_build,v 1.2 2019/10/07 12:50:13 jca Exp $

XXX is lld-7.0.1 being too picky here?

  ld: error: duplicate symbol 'pdb_search_init' in version script

Work around by spelling out the affected symbols.

Index: source3/wscript_build
--- source3/wscript_build.orig
+++ source3/wscript_build
@@ -135,7 +135,9 @@ ldapsam_pdb_match = ['!priv2ld', '!smbldap_search_doma
                      '!get_attr_list']
 private_pdb_match.append('!pdb_nds_*')
 private_pdb_match.append('!pdb_ldapsam_init*')
-private_pdb_match.append('!pdb_*_init')
+private_pdb_match.append('!pdb_samba_dsdb_init')
+private_pdb_match.append('!pdb_smbpasswd_init')
+private_pdb_match.append('!pdb_tdbsam_init')
 private_pdb_match = private_pdb_match + ldapsam_pdb_match
 
 bld.SAMBA3_LIBRARY('samba-passdb',
